hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/SplitWALProcedure.java

```diff
@@ -51,6 +51,7 @@ public class SplitWALProcedure
   private ServerName worker;
   private ServerName crashedServer;
   private RetryCounter retryCounter;
+  private Integer workerChangeCount = 0;
```

Comment:
Why Integer not int?

Reply:
I might have wrongly remembered that becuase of autoboxing and unboxing Interger is better for performance. I have changed it to int